events/tests.py

```diff
@@ -21,6 +25,71 @@ def test_can_store_employee(self):
 
         self.assertEqual(Event.objects.first().user, employee)
 
+
+class PeriodModelTestCase(TestCase):
+
+    def test_create(self):
+        employee = EmployeeFactory()
+        event1 = Event.objects.create(user=employee, time=timezone.now())
+        event2 = Event.objects.create(user=employee, time=timezone.now())
+
+        self.assertEqual(Period.objects.count(), 1)
+        period = Period.objects.last()
+        self.assertEqual(period.user, employee)
+        self.assertEqual(period.start, event1)
+        self.assertEqual(period.end, event2)
+
+    def test_can_be_completed(self):
+        PeriodFactory(completed=True)
+
+        self.assertTrue(Period.objects.last().completed)
+
+    def test_can_be_billable(self):
+        PeriodFactory(billable=True)
+
+        self.assertTrue(Period.objects.last().billable)
+
+    def test_event_occurred_start(self):
+        employee = EmployeeFactory()
+        event = EventFactory(user=employee)
+
+        Period.objects.event_occurred(event)
```

Comment:
You're calling this directly here and in other tests.  I can't seem to find any test that requires new events to call Period.objects.event_occurred
It seems like there should be a test that posts events to /events/new back to back